README.md

```diff
@@ -18,7 +18,7 @@ The server software that powers the XRP Ledger is called `rippled` and is availa
 
 - **[Censorship-Resistant Transaction Processing][]:** No single party decides which transactions succeed or fail, and no one can "roll back" a transaction after it completes. As long as those who choose to participate in the network keep it healthy, they can settle transactions in seconds.
 - **[Fast, Efficient Consensus Algorithm][]:** The XRP Ledger's consensus algorithm settles transactions in 4 to 5 seconds, processing at a throughput of up to 1500 transactions per second. These properties put XRP at least an order of magnitude ahead of other top digital assets.
-- **[Finite XRP Supply][]:** When the XRP Ledger began, 100 billion XRP were created, and no more XRP will ever be created. (Each XRP is subdivisible down to 6 decimal places, for a grand total of 100 quintillion _drops_ of XRP.) The available supply of XRP decreases slowly over time as small amounts are destroyed to pay transaction costs.
+- **[Finite XRP Supply][]:** When the XRP Ledger began, 100 billion XRP were created, and no more XRP will ever be created. (Each XRP is subdivisible down to 6 decimal places, for a grand total of 100 quadrillion _drops_ of XRP.) The available supply of XRP decreases slowly over time as small amounts are destroyed to pay transaction costs.
```

Comment:
A couple of comments:

>Each XRP is subdivisible down to 6 decimal places, for a grand total of 100 quadrillion _drops_ of XRP.

I don't like this wording. A _drop_ is a unit the same way a cent is. You'd never say: "I withdrew 100,000 cents of dollar."

We should have something like this instead:

>Each XRP is subdivisible to 1,000,000 _drops_ for a grand total of 100 quadrillion _drops_ of XRP.

Furthermore, it occurs to me that this parenthetical just confuses things unnecessarily and in this context may not anything to the conversation. What does it matter how many quadrillion drops there are? The supply of XRP is accounted for in XRP in every place where a human is liable to look (except in the ledger internals, and I think that distinction belongs in the documentation and not the general README file).

Reply:
That's fair. I would be fine with removing the parenthetical entirely here.